lib/customer.rb

```diff
@@ -0,0 +1,50 @@
+require 'csv'
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
We technically don't need to give `:address` `attr_accessor` -- it can be in `attr_reader`.

We'd need `attr_accessor` if we ever called `customer.address = ...` _outside_ of this class. You end up modifying `address`, but you do so within the class. :)